src/app/utils/externalLinks.ts

```diff
@@ -1,14 +1,37 @@
 import { Layer } from '../../oasis-nexus/api'
 
-export const socialMedia = {
-  telegram: 'https://t.me/oasisprotocolcommunity',
-  twitter: 'https://twitter.com/oasisprotocol',
-  discord: 'https://oasis.io/discord',
-  // This API link is for testing if invite is still valid.
-  isDiscordStillValid: 'https://oasis.io/discord/invite-api-check',
-  youtube: 'https://www.youtube.com/channel/UC35UFPcZ2F1wjPxhPrSsESQ',
-  reddit: 'https://www.reddit.com/r/oasisnetwork/',
-}
+const customTelegram = process.env.REACT_APP_SOCIAL_TELEGRAM
+const customTwitter = process.env.REACT_APP_SOCIAL_TWITTER
+const customDiscord = process.env.REACT_APP_SOCIAL_DISCORD
+const customYouTube = process.env.REACT_APP_SOCIAL_YOUTUBE
+const customReddit = process.env.REACT_APP_SOCIAL_REDDIT
+const customLinkedin = process.env.REACT_APP_SOCIAL_LINKEDIN
+const customDocs = process.env.REACT_APP_SOCIAL_DOCS
+const customHome = process.env.REACT_APP_SOCIAL_HOME
+
+const hasCustomSocialLinks =
+  !!customTelegram || !!customTwitter || !!customDiscord || !!customYouTube || !!customReddit
+
+export const socialMedia = hasCustomSocialLinks
+  ? {
+      telegram: customTelegram,
+      twitter: customTwitter,
+      discord: customDiscord,
+      youtube: customYouTube,
+      reddit: customReddit,
+      linkedin: customLinkedin,
+      docs: customDocs,
+      home: customHome,
+    }
+  : {
+      telegram: 'https://t.me/oasisprotocolcommunity',
+      twitter: 'https://twitter.com/oasisprotocol',
+      discord: 'https://oasis.io/discord',
+      // This API link is for testing if invite is still valid.
+      isDiscordStillValid: 'https://oasis.io/discord/invite-api-check',
+      youtube: 'https://www.youtube.com/channel/UC35UFPcZ2F1wjPxhPrSsESQ',
+      reddit: 'https://www.reddit.com/r/oasisnetwork/',
+    }
```

Comment:
This should default to Oasis in `.env` file.

Reply:
Yes, now we have the oasis links in the `.env` file.